app/javascript/dashboard/components/ChatList.vue

```diff
@@ -507,16 +522,25 @@ export default {
     },
     label() {
       this.resetAndFetchData();
+      this.updateVirtualListProps('label', this.label);
     },
     conversationType() {
       this.resetAndFetchData();
+      this.updateVirtualListProps('conversationType', this.conversationType);
     },
     activeFolder() {
       this.resetAndFetchData();
+      this.updateVirtualListProps('foldersId', this.foldersId);
     },
     chatLists() {
       this.chatsOnView = this.conversationList;
     },
+    showAssigneeInConversationCard(newVal) {
+      this.virtualListExtraProps = {
+        ...this.virtualListExtraProps,
+        showAssignee: newVal,
+      };
```

Comment:
We can use `updateVirtualListProps('showAssignee', newVal)` here

Reply:
Missed it